.gitignore

```diff
@@ -10,6 +10,8 @@ __pycache__/
 # Distribution / packaging
 .Python
 env/
+venv/
```

Comment:
are you using pyvenv vs virtualenv?
isn't this contained in docker? or you got outside of docker :8ball: